docs/samples/Local TXDC Setup.md

```diff
@@ -64,10 +64,9 @@ helm install plato charts/tractusx-connector \
     --namespace cx \
     --create-namespace \
     --set fullnameOverride=plato \
-    --set controlplane.image.tag=0.2.0 \
+    --set controlplane.image.tag=latest \
```

Comment:
using `latest` is dangerous, because the `latest` tag gets overwritten on every build we do on tractusx-edc